sklearn/linear_model/_perceptron.py

```diff
@@ -224,3 +224,12 @@ def __init__(
             class_weight=class_weight,
             n_jobs=n_jobs,
         )
+
+    def __sklearn_tags__(self):
+        tags = super().__sklearn_tags__()
+        tags._xfail_checks = {
+            "check_sample_weights_invariance": (
+                "sample_weight is not equivalent to removing/repeating samples."
+            ),
```

Comment:
This one is probably a consequence of a non-deterministic fit combined with a non-convex loss.